store/tikv/2pc.go

```diff
@@ -759,7 +766,7 @@ func (action actionPessimisticLock) handleSingleBatch(c *twoPhaseCommitter, bo *
 		}
 		// Because we already waited on tikv, no need to Backoff here.
 		// tikv default will wait 3s(also the maximum wait value) when lock error occurs
-		_, err = c.store.lockResolver.ResolveLocks(bo, c.startTS, locks)
+		_, _, err = c.store.lockResolver.ResolveLocks(bo, c.startTS, locks, nil)
```

Comment:
We better set TS to 0 for PessimisticLock.

Reply:
Done